modules/Data.py

```diff
@@ -141,3 +141,14 @@ def truncate(f, n):
         return float('{0:.{1}f}'.format(f, n))
     i, p, d = s.partition('.')
     return float('.'.join([i, (d + '0' * n)[:n]]))
+
+
+# gets the git commit count as version for master.
+def get_bot_version():
+    import subprocess
+    output = subprocess.check_output(["git", "rev-list","--count", "HEAD"])
+    try:
+        int(output)
+        return output
+    except ValueError:
```

Comment:
You need to catch 'Exception' here I think. Or work out the other exceptions. 

If it's not a git repo, output won't exist and it'll be a NameError, if git doesn't exist it'll be an OSError. Not sure what else there could be.

Reply:
Fixed.